sql/binding_contexts.cc

```diff
@@ -169,13 +169,24 @@ doGetFunction(const Utf8String & tableName,
 
     auto fn = outer.doGetColumnFunction(functionName);
 
-    return {[=] (const std::vector<ExpressionValue> & args,
+    if (fn)
+    {
+         return {[=] (const std::vector<ExpressionValue> & args,
                  const SqlRowScope & context)
             {
                 auto & col = static_cast<const ColumnContext &>(context);
                 return fn(col.columnName, args);
             },
             std::make_shared<Utf8StringValueInfo>()};
+    }
+
+    auto sqlfn = SqlBindingScope::doGetFunction(tableName, functionName, args);
```

Comment:
Will this break training code like svd which do not support operation on columns but support column expression?